librad/src/identities/payload.rs

```diff
@@ -655,9 +655,9 @@ mod tests {
         }
     }
 
-    fn gen_user_payload() -> impl Strategy<Value = UserPayload> {
-        (any::<User>(), proptest::option::of(any::<UpstreamUser>())).prop_map(|(user, up)| {
-            let mut u = UserPayload::new(user);
+    fn gen_user_payload() -> impl Strategy<Value = PersonPayload> {
```

Comment:
Dunno how far you want to go with renaming, but the function names still mention `user`
```suggestion
    fn gen_person_payload() -> impl Strategy<Value = PersonPayload> {
```